src/components/board/Stack.vue

```diff
@@ -42,11 +42,14 @@
 						value="">
 				</form>
 			</transition>
-			<Actions v-if="canManage && !isArchived" :force-menu="true">
-				<ActionButton icon="icon-archive" @click="modalArchivAllCardsShow=true">
+			<Actions v-if="!isArchived" :force-menu="true">
+				<ActionButton v-if="canManage" icon="icon-archive" @click="modalArchivAllCardsShow=true">
```

Comment:
Archiving all cards is something that would only require edit permissions, maybe you can fix this in the same run.